tensorflow/python/ops/math_ops.py

```diff
@@ -920,6 +921,39 @@ def reduce_any(input_tensor, reduction_indices=None, keep_dims=False,
                                                         reduction_indices),
                            keep_dims, name=name)
 
+def trace(x, name=None):
+  """ Compte the trace of a tensor `x`.
+
+  `trace(x)` returns the sum of along the diagonal.
+  
+  For example:
+
+  ```python
+  # 'x' is [[1, 1],
+  #         [1, 1]]
+  tf.trace(x) ==> 2
+  
+  # 'x' is [[1,2,3],
+  #         [4,5,6],
+  #         [7,8,9]]
+  tf.trace(x) ==> 15
+  ```
+
+  Args:
+    input_tensor: Rank 2 tensor.
```

Comment:
Can you instead write "input_tensor: 2-D tensor." to be consistent with other ops with dimensionality constraints?